AlphaWallet/Settings/ViewControllers/ExportJsonKeystorePasswordViewController.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  ExportJsonKeystorePasswordViewController.swift
+//  AlphaWallet
+//
+//  Created by Jerome Chan on 1/12/21.
+//
+
+import UIKit
+
+class ExportJsonKeystorePasswordViewController: UIViewController {
```

Comment:
This should have a Done button instead of Return and tapping the button should hide the keyboard